guides/spin/pipeline-templates/index.md

```diff
@@ -90,10 +90,10 @@ spin pipeline-templates get --id myPipelineTemplate
 }
 ```
 
-### Visualize a hydrated pipeline template with `plan`
+### Visualize a hydrated pipeline with `plan`
 
 ```bash
-spin pipeline-templates plan --config <path to template config>
+spin pipeline-templates plan --config <path to pipeline config>
```

Comment:
This should change to `--file` instead of `--config` for the newest spin release